sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/UnusedPrivateMember.cs

```diff
@@ -100,7 +100,7 @@ protected override void Initialize(SonarAnalysisContext context)
                                 c.Compilation.GetSemanticModel,
                                 removableSymbolsCollector.PrivateSymbols.Select(s => s.Name).ToHashSet());
 
-                            if (!VisitDeclaringReferences(namedType, usageCollector, c.Compilation))
+                            if (!VisitDeclaringReferences(namedType, usageCollector, c.Compilation, includeGeneratedFile: true))

```

Comment:
... But we do want to collect the usages of other members even in generated files.